edward/Achrismastree/src/main/java/foo/Foo.java

```diff
@@ -0,0 +1,79 @@
+package foo;
+import java.util.Scanner;
+
+public class Foo {
+    public static void main(String[] args){
+        System.out.println("Enter a positive integer number");
+        Scanner number_receiver = new Scanner(System.in);
+
+        int y = 0;
+        int x = number_receiver.nextInt();
+
+        if(x < 1){
+            System.out.println("!@#*@*&$*@&$*@(!))@$&*(@$&*!&(@%@)$&**(@!");
+            return;
+        }
+        while(0 == 0) {

```

Comment:
Usually more common way (and easier to ready) to do loop like this will be : 

`for (y=0; y < x; x++)
`